.github/workflows/deploy.yml

```diff
@@ -0,0 +1,19 @@
+name: Deploy branch
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  deployer:
+    runs-on: ubuntu-latest
```

Comment:
Please also setup timeout to avoid draining our spendings limits:
https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes
I suggest: `timeout-minutes: 30`